lib/streamlit/elements/vega_charts.py

```diff
@@ -95,6 +96,14 @@
 }
 
 VegaLiteSpec: TypeAlias = "dict[str, Any]"
+AltairChart: TypeAlias = """Union[
+    alt.Chart,
+    alt.LayerChart,
+    alt.ConcatChart,
+    alt.HConcatChart,
+    alt.VConcatChart,
+    alt.FacetChart,
```

Comment:
nit: there is also an `alt.RepeatChart` type that might be relevant to add here.